packages/@o3r/create/src/index.ts

```diff
@@ -198,6 +198,19 @@ const prepareWorkspace = (relativeDirectory = '.', projectPackageManager = 'npm'
     }));
   }
 
+  if (argv.registry) {
```

Comment:
That option is available on all npm create

Reply:
Then do we receive it?
Because what you describe seems to be the difference between `npm create <package> --registry` and `npm create <package> -- --registry`

Reply:
As discussed for npm we need to relate to the env variable `npm_config_registry`